### DISCUSSION TITLE
HTTP URL is invalid

### DISCUSSION THREAD

**Discussion by piyu5h** [provectus/kafka-ui#2535] [Q&A] [answered]

**Comment by Haarolean** [comment, marked as answer]
Hi, is the fact that it's edited and empty now means it's solved?

**Reply by piyu5h** [reply]
Yes, It solved.
I am getting this for schema registry, though I checked URL is valid and perfectly working.
I running kafka-ui from Docker and Schema Registry is also hosted on same docker host. Even it is showing same for Confluent cloud Schema registry URl. What I am doing wrong here ?

[ http://192.168.10.5:8081/subjects] is not a valid HTTP URL

**Reply by Starkie** [reply]
Hi @piyu5h, maybe you found the solution already, but for future reference: **There's a space right before the URL**.

Instead of:

```yaml
KAFKA_CLUSTERS_0_SCHEMAREGISTRY: ' http://192.168.10.5:8081/subjects'
```
It should be:
```yaml
KAFKA_CLUSTERS_0_SCHEMAREGISTRY: 'http://192.168.10.5:8081/subjects'
```
Just spent like half an hour scratching my head over this lol

**Reply by Haarolean** [reply]
@Starkie that's like the third time I encounter people doing the same thing: leaving an extra space before the SR URL %)

**Reply by Starkie** [reply]
The dangers of copy pasting stuff around the docker file, I guess 😅
